spec/unit/packagers/ips_spec.rb

```diff
@@ -171,11 +170,11 @@ module Omnibus
         expect(manifest_file_contents).to include("set name=variant.arch value=i386")
       end
 
-      context "when symlinks.erb exists" do
+      context "when symlinks_file exists" do
```

Comment:
shouldn't we have another case to test now? It looks like `#{safe_base_package_name}-symlinks.erb"` is tested here, but there could still be `symlinks.erb`?